app/ping/__init__.py

```diff
@@ -2,5 +2,7 @@
 
 from app.ping.views import ping
 
+from app.ping.judge_endpoint.endpoint import judge_endpoint_bp
 
-__all__ = ["ping"]
+
+__all__ = ["ping", "judge_endpoint_bp"]
```

Comment:
```suggestion
__all__ = ["ping"]
```

No va acá, sino dentro del init del judge_endpoint (carpeta)